src/geometry/geom2/applyTransforms.js

```diff
@@ -14,8 +14,8 @@ const applyTransforms = (geometry) => {
 
   // apply transforms to each side
   geometry.sides = geometry.sides.map((side) => {
-    const p0 = vec2.canonicalize(vec2.transform(geometry.transforms, side[0]))
-    const p1 = vec2.canonicalize(vec2.transform(geometry.transforms, side[1]))
+    const p0 = vec2.transform(geometry.transforms, side[0])
```

Comment:
bye bye canonicalize ? :)

Reply:
Agreed. Nice idea but doesn’t really solve the issues with floating point numbers. We NOW have a whole set of awesome tools to work around those issues. We just need to use them appropriately.